### PR TITLE
ENYO-5632: Add core-js as a dev dependency for local installs

### PR DESCRIPTION
**Pull request by ryanjduffy**
## Issue
When installing cli locally (e.g. via `enact create --locale test`), the babel polyfills can fail to be found if another package (e.g. `react`) has a dependency on a lower version of `core-js`.

## Resolution
Explicitly add the `core-js@2` dependency when using the `--local` option to ensure we have a compatible set of polyfills.

See: https://babeljs.io/docs/en/next/babel-preset-env.html#usebuiltins

### CODE REVIEW

**Comment by ryanjduffy** [comment]
:+1: Fixed up!

File: commands/create.js [hunk 1]
```diff
@@ -258,7 +259,9 @@ function api(opts = {}) {
 			.then(() => {
 				if (opts.local) {
 					console.log('Installing @enact/cli locally. This might take a couple minutes.');
-					return npmInstall(opts.directory, opts.verbose, '--save-dev', ENACT_DEV_NPM);
+					return npmInstall(opts.directory, opts.verbose, '--save', CORE_JS_NPM).then(() =>
```
Comment (webOS101) [inline commands/create.js]: Any reason to not just rename `ENACT_DEV_NPM` to `ENACT_LOCAL_DEPS` and just combine them together into one variable and call?

Comment (JayCanuck) [reply commands/create.js]: `CORE_JS_NPM` would be a dependency and ENACT_DEV_NPM would be a devDependency.  Don't believe you can use both `--save` and `--save-dev` together in 1 command without one overriding the other.

Comment (webOS101) [reply commands/create.js]: oh.  :nothingtodohere:

Comment (ryanjduffy) [reply commands/create.js]: As it stands, they're landing in different spots (dependencies vs devDependencies) so I don't think that'd be compatible.

**Pull request merged**